### ISSUE TITLE
ARIA-tools version should be pinned to tag

### ISSUE THREAD

**Issue by alexdunncs** [aria-jpl/mintpy-pge#22]
Currently, `dev` branch is used as old release broke due to change in third-party dependency.
Once a new tag is released, the Dockerfile should be changed to check out that tag.
